### PR TITLE
update snapshot download

### PR DESCRIPTION
**Pull request by dustinxie**

### CODE REVIEW

File: archive-node.md [hunk 1]
```diff
@@ -57,10 +58,78 @@ curl https://raw.githubusercontent.com/iotexproject/iotex-bootstrap/master/confi
 curl https://raw.githubusercontent.com/iotexproject/iotex-bootstrap/master/genesis_mainnet.yaml > $IOTEX_HOME/etc/genesis.yaml
 curl https://raw.githubusercontent.com/iotexproject/iotex-bootstrap/master/trie.db.patch > $IOTEX_HOME/data/trie.db.patch
 curl https://storage.googleapis.com/blockchain-golden/poll.mainnet.db > $IOTEX_HOME/data/poll.db
-
-#download snapshot and unzip data
-// TODO: add download link
 ```
+## <a name="download"/>Download Data
```
Comment (guo) [inline archive-node.md:62]: don't use a tag for a whole paragraph

File: archive-node.md [hunk 2]
```diff
@@ -34,7 +35,7 @@ source ~/.bashrc
 go version
 ```
 
-## <a name="prepdata"/>Prepare Data
+## <a name="prephome"/>Prepare Home Directory
```
Comment (guo) [inline archive-node.md:38]: same here

File: archive-node.md (hunk 1 above)
Comment (guo) [reply archive-node.md:62]: oh. you forgot to close the a tag.

Comment (dustinxie) [reply archive-node.md:62]: this is how we set a clickable text/tag on the page, so you can click the text and navigate to the the corresponding paragraph. Same as `README.md`
or what is the usage you recommend?

**Review by Liuhaai — APPROVED** [review]

**Review by raullenchai — APPROVED** [review]

**Review by envestcc — APPROVED** [review]

**Pull request merged**